geopandas/sindex.py

```diff
@@ -73,8 +70,8 @@ class RTreeIndex(rtree.index.Index):
 
         Parameters
         ----------
-        geometry : GeoSeries
-            GeoSeries from which to build the spatial index.
+        geometry : np.array of PyGEOS geometries
```

Comment:
actually shapely geometries instead of pygeos?

Reply:
yep good catch